core/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -159,13 +175,22 @@ public static IndicesOptions fromMap(Map<String, Object> map, IndicesOptions def
      * false otherwise
      */
     public static boolean isIndicesOptions(String name) {
-        return "expand_wildcards".equals(name) || "expandWildcards".equals(name) ||
-                "ignore_unavailable".equals(name) || "ignoreUnavailable".equals(name) ||
-                "allow_no_indices".equals(name) || "allowNoIndices".equals(name);
+        switch (name) {
+            case "expand_wildcards":
+            case "expandWildcards":
+            case "ignore_unavailable":
+            case "ignoreUnavailable":
+            case "allow_no_indices":
+            case "allowNoIndices":
+            case "include_hidden":
```

Comment:
man I don't wanna introduce more broken settings here it's enough that we have the others ?